server/services/offender/index.js

```diff
@@ -33,7 +33,7 @@ const createOffenderService = (
       );
       logger.debug(e.stack);
       return {
-        error: 'We are not able to show your incentives summary at this time',
+        error: 'We are not able to show your incentive summary at this time',
```

Comment:
Also this I'd add level just to be absolutely clear.

Reply:
@LucasCairns just this one I think?

Reply:
updated 👍